packages/smart-wallet/src/smartWallet.js

```diff
@@ -164,6 +211,16 @@ export const prepareSmartWallet = (baggage, shared) => {
 
   const makeRecorderKit = prepareRecorderKit(baggage, shared.publicMarshaller);
 
+  const BRAND_TO_PURSES_KEY = 'brandToPurses';
+  const walletPurses = provide(baggage, BRAND_TO_PURSES_KEY, _k => {
```

Comment:
style: I find the "k" distracting. it's not informing interpretation of this code. consider, `_` or `()`

Reply:
Really? I find it kinda useful as a reminder of what it is that I'm ignoring. But I don't feel that strongly about it.

Reply:
I was going to go with `_` but it collides with an earlier use of `_`. `()` feels wrong but I guess I can deal.